### ISSUE TITLE
Uploaded file names wasn't transliterated

### ISSUE THREAD

**Issue by Al-Rozhkov** [backdrop-contrib/file_resup#4]
Transliteration module was merged to Backdrop core, but there isn't Transliteration module anymore. This module had additional check `module_exists('transliteration')`, therefore uploaded files wasn't transliterated.

PR: https://github.com/backdrop-contrib/file_resup/pull/3

**Comment by klonos** [comment]
Good catch @Al-Rozhkov 👍 ...might be worth checking the core transliteration settings available in `/admin/config/media/file-system` to see if transliteration is enabled (`file_transliterate_uploads` in `system.core.json`).

**Comment by klonos** [comment]
...basically add `config_get('system.core', 'file_transliterate_uploads')` in that if statement.

**Comment by Al-Rozhkov** [comment]
@klonos I agree. But file names wasn't transliterated for some reason. Needs more debugging.
